utilities_help/createToolboxMLTBX.m

```diff
@@ -5,7 +5,8 @@ function createToolboxMLTBX(verNumber)
 % project root.
 % Input:
 %   verNumber- string in major.minor.bugfix format
-prjFileName=fullfile("fsdaToolboxPackaging.prj");
+% Copyright 2023 The MathWorks, Inc.
+prjFileName="fsdaToolboxPackaging.prj";
 toolboxOption=matlab.addons.toolbox.ToolboxOptions(prjFileName);
```

Comment:
The name of the variable should not be toolboxOption to avoid confusion with the API name, make it more descriptive of what information it holds

Reply:
Replaced toolboxOption with packagingData.